pennylane/qchem/tapering.py

```diff
@@ -427,7 +436,7 @@ def optimal_sector(qubit_op, generators, active_electrons):
 
     perm = []
     for tau in generators:
-        symmstr = np.array([1 if wire in tau.ops[0].wires else 0 for wire in qubit_op.wires])
+        symmstr = np.array([1 if wire in tau.wires else 0 for wire in qubit_op.wires])
```

Comment:
It seems that using `enable_new_opmath()` breaks this function. 

```
    enable_new_opmath()
    hamiltonian_op = pauli_sentence(hamiltonian).operation()
    new_generators = qml.qchem.symmetry_generators(hamiltonian_op)
    new_paulixops = qml.qchem.paulix_ops(new_generators, qubits)
    new_paulix_sector = qml.qchem.optimal_sector(hamiltonian_op, new_generators, mol.n_electrons)
    disable_new_opmath() 
```

Look at the following output for H2 for the above code snippet.

```
Generators: 
  (1.0) [Z0 Z1]
  (1.0) [Z0 Z2]
  (1.0) [Z0 Z3]
(((Identity(wires=[0]) @ PauliZ(wires=[0])) @ PauliZ(wires=[1])) @ Identity(wires=[2])) @ Identity(wires=[3])
(((Identity(wires=[0]) @ PauliZ(wires=[0])) @ Identity(wires=[1])) @ PauliZ(wires=[2])) @ Identity(wires=[3])
(((Identity(wires=[0]) @ PauliZ(wires=[0])) @ Identity(wires=[1])) @ Identity(wires=[2])) @ PauliZ(wires=[3])
PauliXops:  [PauliX(wires=[1]), PauliX(wires=[2]), PauliX(wires=[3])] [PauliX(wires=[1]), PauliX(wires=[2]), PauliX(wires=[3])]
PauliXsec:  [1, -1, -1] [1, 1, 1]
```

Reply:
Fixed 👍🏼

Reply:
What was causing the bug though? 🤔